src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+
+        for (Ball ball : balls) {
+            ball = lottery.getRandomBall();
```

Comment:
you dont use this array actually
```suggestion
        for (int i = 0; i < BALLS_TO_CREATE; i++) {
            Ball ball = lottery.getRandomBall();
```